arm-web/2016-09-01/swagger/AppServiceEnvironments.json

```diff
@@ -1861,6 +1861,250 @@
     },
     "AppServiceEnvironment": {
```

Comment:
please check properties sharing described above can be applied for AppServiceEnvironment and AppServiceEnvironmentResource